feature-libs/cart/base/components/clear-cart/clear-cart-dialog/clear-cart-dialog-component.service.ts

```diff
@@ -1,77 +1,82 @@
 import { Injectable } from '@angular/core';
-import { GlobalMessageService, GlobalMessageType } from '@spartacus/core';
-import { ActiveCartFacade } from '@spartacus/cart/base/root';
-import { filter, switchMap, take, tap } from 'rxjs/operators';
-import { BehaviorSubject, Observable } from 'rxjs';
+import {
+  EventService,
+  GlobalMessageService,
+  GlobalMessageType,
+  UserIdService,
+} from '@spartacus/core';
+import {
+  ActiveCartFacade,
+  MultiCartFacade,
+  DeleteCartSuccessEvent,
+  DeleteCartFailEvent,
+} from '@spartacus/cart/base/root';
+import { mapTo, switchMap, take, tap, withLatestFrom } from 'rxjs/operators';
 import { LaunchDialogService } from '@spartacus/storefront';
+import { merge } from 'rxjs';
 
 @Injectable({
   providedIn: 'root',
 })
 export class ClearCartDialogComponentService {
-  isClearing$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
-
   constructor(
     protected launchDialogService: LaunchDialogService,
     protected globalMessageService: GlobalMessageService,
-    protected activeCartFacade: ActiveCartFacade
+    protected activeCartFacade: ActiveCartFacade,
+    protected multiCartFacade: MultiCartFacade,
+    protected userIdService: UserIdService,
+    protected eventService: EventService
   ) {}
 
+  /**
+   * Clear all entries and info of active cart
```

Comment:
Perhaps update the doc comment

Reply:
ooups... this was my new doc for the method after using delete cart action. Before it was `Clear all entries of active cart`